domains/client/domain-executor/src/fraud_proof.rs

```diff
@@ -128,13 +128,13 @@ where
                 parent_header.hash(),
                 Default::default(),
             );
-            let execution_phase = ExecutionPhase::InitializeBlock {
-                call_data: new_header.encode(),
-            };
+            let execution_phase = ExecutionPhase::InitializeBlock;
+            let initialize_block_call_data = new_header.encode();
```

Comment:
The `new_header` is mock, I'm curious about why use a mock header here.

Reply:
Sure! I will update this test with the new testing framework to cover the `initialize_block` case.